examples/lead-generation/src/graph/graph.ts

```diff
@@ -0,0 +1,41 @@
+import { CSV_GENERATOR_NODE_NAME, csvGeneratorNode } from "@/graph/nodes/csv-generator-node";
+import { ERROR_HANDLER, errorHandlerNode } from "@/graph/nodes/error-handler-node";
+import { OUTREACH_MESSAGE_NODE_NAME, outreachMessageNode } from "@/graph/nodes/outreach-message-node";
+import { ENRICH_THERAPISTS_NODE_NAME, enrichTherapistNode } from "@/graph/nodes/therapist-enrichment-node";
+import { FETCH_THERAPISTS, fetchTherapistsNode } from "@/graph/nodes/therapist-fetcher-node";
+import { URL_VALIDATOR, urlValidatorNode, validUrlCounterEdge } from "@/graph/nodes/url-validator-node";
+import { state } from "@/graph/state";
+import { END, START, StateGraph } from "@langchain/langgraph";
+
+export type LeadGenerationGraphResult = {
+  csvContent: string;
+  csvPath: string;
+  error: string;
+};
+
+export const leadGenerationGraph = async (graphInputs: string[]): Promise<LeadGenerationGraphResult> => {
+  const graphBuilder = new StateGraph(state)
+    .addNode(URL_VALIDATOR, urlValidatorNode)
+    .addNode(FETCH_THERAPISTS, fetchTherapistsNode)
```

Comment:
Nit: Can we add the `NODE_NAME` suffix on these nodes to stay consistent with the other nodes. Same with `ERROR_HANDLER` below too.